amf-cli/shared/src/test/resources/semantic-jsonschema/dialects/allOf-complex.yaml

```diff
@@ -25,9 +25,9 @@ nodeMappings:
     mapping:
       a:
         range: any
-        mandatory: false
         enum:
           - condition
+        mandatory: false
```

Comment:
weird annotation change

Reply:
? It only changes the position. It is because I needed to change the order of the in-memory mapping field creation (now the minCount/mandatory field are populated before or after, depending on it is an array or not)